helpers/eventwebhook/eventwebhook_test.go

```diff
@@ -0,0 +1,101 @@
+package eventwebhook
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	timestamp      = "1588788367"
+	testPublicKey  = "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEEDr2LjtURuePQzplybdC+u4CwrqDqBaWjcMMsTbhdbcwHBcepxo7yAQGhHPTnlvFYPAZFceEu/1FwCM/QmGUhA=="
+	testPrivateKey = "MHcCAQEEIEQmZgBEh9DcU9zMl34czK3xov4AYvm9P3r7rNB2dNXtoAoGCCqGSM49AwEHoUQDQgAEEDr2LjtURuePQzplybdC+u4CwrqDqBaWjcMMsTbhdbcwHBcepxo7yAQGhHPTnlvFYPAZFceEu/1FwCM/QmGUhA=="
+	signature      = "MEUCIQCtIHJeH93Y+qpYeWrySphQgpNGNr/U+UyUlBkU6n7RAwIgJTz2C+8a8xonZGi6BpSzoQsbVRamr2nlxFDWYNH2j/0="
+)
+
+func generateTestPayload() []byte {
+	payload, _ := json.Marshal(map[string]interface{}{
```

Comment:
we should check the error here instead of swallowing it

Reply:
I think for unittests it would be fine since we are using golang's internal encoding/json library.